src/pages/PetitionPage/PetitionPage.tsx

```diff
@@ -22,23 +23,32 @@ function PetitionPage() {
     });
   };
   const observer = new IntersectionObserver(callback, {
-    rootMargin: '100px 100px',
+    rootMargin: '-100px -100px',
   });
 
   useEffect(() => {
-    observer.observe(document.querySelector<any>('.fake'));
+    observer.observe(document.querySelector<any>('h1'));
+    return () => {
+      observer.disconnect();
+    };
```

Comment:
위의 리뷰처럼 고친다면 이부분도
```suggestion
    intersectionObserverRef.current.observe(document.querySelector<any>('h1'));
    return () => {
      intersectionObserverRef.current.disconnect();
    };
```
처럼 될것 같습니다! 그리고 가능하다면 쿼리셀렉터를 통해 dom을 얻어오기보다는 직접 해당돔에 ref를 달아서 얻어오는것이

Reply:
```  
const titleRef = useRef<any>(null);
```
titleRef의 형식을 뭘로 해야할지 몰라서 일단 any로 했는데 뭐로 하면 좋을까요?
이것저것 다 해봤는데 다 안되네요 ㅎㅅㅎ..